packages/core/src/layout/TabbedCard/TabbedCard.stories.tsx

```diff
@@ -15,19 +15,37 @@
  */
 import React, { useState } from 'react';
 import { TabbedCard, CardTab } from '.';
+import { Grid } from '@material-ui/core';
+
+const cardContentStyle = { height: 200, width: 500 };
 
 export default {
   title: 'Tabbed Card',
   component: TabbedCard,
+  decorators: [
```

Comment:
Oh, that's interesting. Perhaps we want to add this to the global scheme?

Reply:
That could be a good idea unless there are exceptions to components that should be wrapped in a grid.

Reply:
Typically all components are laid in some form of grid inside Plugins. Should be fine to add to all. In a separate PR though.